pkg/ccl/backupccl/schedule_pts_chaining_test.go

```diff
@@ -293,18 +286,6 @@ INSERT INTO t values (1), (10), (100);
 			hlc.Timestamp{WallTime: backupAsOfTimes[0].Round(time.Microsecond).UnixNano()})
 
 		th.sqlDB.Exec(t, `DROP SCHEDULE $1`, fullID)
-
-		// Run the inc schedule.
```

Comment:
we should certainly check in this case the that incermental pts record was dropped

Reply:
making sure you saw this!

Reply:
Currently I just changed it so when it tries to load the scheduled job it expects an error

Reply:
i think you should check that the PTS has been dropped. E.g. you could run after the inc has been dropped

```
ptsRecord, err = th.server.ExecutorConfig().(sql.ExecutorConfig).ProtectedTimestampProvider.
			GetRecord(context.Background(), txn, id)
```
			
and expect an error (i pulled this from the checkPTSRecord function)

Reply:
looks good! resolving.